runtime/runtime.go

```diff
@@ -3521,3 +3543,11 @@ func hash(
 
 	return interpreter.ByteSliceToByteArrayValue(inter, result)
 }
+
+func alwaysValidates(
```

Comment:
This function actually 'neverValidates'? 🤔  Maybe rename it to something like `noOp`?

Reply:
Haha, naming is hard. I guess the intention was here "the result is always valid", but you are right, the function does not perform any validation, it could be misunderstood. I do not have a good idea, but would recommend to still include something in the name that expresses the function is for public key validation – the parameters do not indicate it, and the function could get accidentally used for other purposes and its behaviour changed in the future.

Also, the function seems to be duplicated in the test code. Maybe export here and remove the duplicate in the test code.

Reply:
Clarified!